example/src/main/java/com/stripe/example/activity/KlarnaSourceActivity.kt

```diff
@@ -42,11 +42,20 @@ class KlarnaSourceActivity : AppCompatActivity() {
             viewBinding.progressBar.visibility = View.VISIBLE
             createKlarnaSource().observe(this, Observer { result ->
                 viewBinding.progressBar.visibility = View.INVISIBLE
+                val stripeAccount = viewBinding.stripeAccount.text
```

Comment:
can move this inside `is SourceViewModel.SourceResult.Success -> {` block

also you can make it a one-liner:

```
                            stripeAccountId = viewBinding.stripeAccount.text.toString().takeIf {
                                it.isNotBlank()
                            }
```

Reply:
ah I knew there must be a way to do this. Still kinda wish they just had an optional monad but I totally get why they do it this way